typewriter/fixes/base.py

```diff
@@ -637,7 +637,8 @@ def touch_typing_import(self, word, node):
         node : Node
         """
         if word in typing_all:
-            if not does_tree_import('typing', word, node):
+            if not type_by_import_stmt('typing', word, node):
```

Comment:
this line makes sure the changes we made for parentheses in our `fixer_utils` is actually used in the `typing` import case